<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Triangle</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="bLineEdit">
    <property name="geometry">
     <rect>
      <x>106</x>
      <y>10</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="setButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cLineEdit">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>10</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="aLineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QLabel" name="areaResultLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>62</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="areaPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>83</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Area</string>
    </property>
   </widget>
   <widget class="QPushButton" name="anglesPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>83</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Angles</string>
    </property>
   </widget>
   <widget class="QLabel" name="anglesResultLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="perimeterResultLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>62</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="perimeterPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>83</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Perimeter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mediansPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>83</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Medians</string>
    </property>
   </widget>
   <widget class="QLabel" name="mediansResultLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="isRightPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>83</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Is Right</string>
    </property>
   </widget>
   <widget class="QLabel" name="isRightLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>62</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="increasePushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>83</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Increase</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="increaseLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
